### ISSUE TITLE
Make external nature of `Packages` navigation item visible

### ISSUE THREAD

**Issue by pitag-ha** [ocaml/v2.ocaml.org#1308]
This would be a **good first issue** for anyone looking to get started improving and contributing to OCaml.org.

The header contains six navigation items: `OCaml`, `Lean`, `Documentation`, `Packages`, `Community` and `News`. All of them apart from `Packages` navigate inside ocaml.org, whereas `Packages` links to an external page (opam.ocaml.org). I suggest to make that visible by marking `Packages` with an external link icon.

**Comment by patricoferris** [comment]
This seems like a great idea, quite a few people are confused by this -- thanks @pitag-ha :))

@Muskan-sam would you like to work on this as part of #1245? Please let us know either way below.

**Comment by Goodiec** [comment]
Hello @pitag-ha, I am an outreachy applicant interested in working on improving the ocaml.org website. I would love to work on this issue, could it be assigned to me?

**Comment by Muskan-sam** [comment]
yes @patricoferris  , I would like to work on this issue. 
Thank you @patricoferris and @pitag-ha

**Comment by patricoferris** [comment]
Hi both 👋 

Thanks for the eagerness to contribute. @Goodiec would you mind working on https://github.com/ocaml/ocaml.org/issues/1310 instead, I did ask @Muskan-sam not that long ago :))

**Comment by Goodiec** [comment]
OK @patricoferris, thank you!

**Comment by pitag-ha** [comment]
Hi @Muskan-sam, are you still working on this issue? Do you need any help? No pressure! I just want to know how it's going :)

**Comment by esticojerry** [comment]
Hi am Esther , an outreachy applicant interested in working on improving the ocaml.org website. I would love to work on this issue, could it be assigned to me?
Am new to open source and would like to start my journey of contribution.

**Comment by pitag-ha** [comment]
Hi @esticojerry, welcome! :) Let's give @Muskan-sam a chance to answer if they're still working in this; I'd say a day or so. If they don't answer, you can work on it.

**Comment by esticojerry** [comment]
Hi, that's ok. Can i get a new task to work on while waiting.

**Comment by pitag-ha** [comment]
I'd say that since @Muskan-sam still hasn't answered, you can go ahead :)

**Comment by Muskan-sam** [comment]
@pitag-ha I am really sorry for answering late. I was stuck with the issue of setting up the project locally on my system. I am done with that issue now and I have successfully installed ocaml.org .

**Comment by Muskan-sam** [comment]
> Hi @Muskan-sam, are you still working on this issue? Do you need any help? No pressure! I just want to know how it's going :)

@pitag-ha yes I am still working on this issue. Can you give me some more time.

**Comment by Muskan-sam** [comment]
![image](https://user-images.githubusercontent.com/76677032/114100519-0ba00780-98e2-11eb-8f01-89c67ba6b1e9.png)

@pitag-ha Is this fine? should I submit a PR ?

**Comment by pitag-ha** [comment]
@Muskan-sam , that does look good, thanks! Why didn't you answer when I asked you 3 days ago if you were still working on this and 2 days ago when we re-assigned this issue?

@esticojerry , have you started working on this?

**Comment by Muskan-sam** [comment]
> @Muskan-sam , that does look good, thanks! Why didn't you answer when I asked you 3 days ago if you were still working on this and 2 days ago when we re-assigned this issue?

 @pitag-ha  I am really sorry for that. I had issues while setting up WSL and the project on my system. The issue was finally solved yesterday and I could build the project successfully on my system. I have completed the work and will be submitting the PR .

**Comment by Muskan-sam** [comment]
@pitag-ha  I had submitted a PR (#1491) for this issue but it had errors of line ending. I rectified that but every time I try making `commit` it shows more than 5000 files have changed. I have tried all possible ways to rectify that but it seems like there is still some problem. I am unable to work on this issue properly and hence would like to withdraw. 
Could you please re-assign this issue to @esticojerry if she is still will to work on this issue or someone else ? I am really sorry for the inconvenience.

**Comment by pitag-ha** [comment]
@Muskan-sam, ok, don't worry about that :) A couple of general tips about the kind of problems you're having:

- as Gargi suggested:
  > You need to setup git so that you can have Unix-like line endings instead of Windows style line endings
  
  What have you tried to get that git setup right? For example, have you seen the [github docs](https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings) (you can click on `windows` at the top) on that?

-  You said 
   > When you I tried `git reset HEAD~` but that didn't work.

   In general, when you say that something didn't work, it helps if you say in which way it didn't work. For example, did the command succeed but did something different than you expected? If so, what did it do? Or did it not succeed? If so, what was the error it returned? (I'm **not** saying to do that now for the case of `git reset HEAD~`, I'm just trying to help you get more concrete feedback in general in the future :))

**Comment by Muskan-sam** [comment]
> @Muskan-sam, ok, don't worry about that :) A couple of general tips about the kind of problems you're having:
> 
> * as Gargi suggested:
>   > You need to setup git so that you can have Unix-like line endings instead of Windows style line endings
>   
>   
>   What have you tried to get that git setup right? For example, have you seen the [github docs](https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings) (you can click on `windows` at the top) on that?

@pitag-ha  I searched for the line ending problem and found a solution on stack overflow. Then I went through the git configuration documentation and configured my git bash.

> * You said
>   > When you I tried `git reset HEAD~` but that didn't work.
>   
>   
>   In general, when you say that something didn't work, it helps if you say in which way it didn't work. For example, did the command succeed but did something different than you expected? If so, what did it do? Or did it not succeed? If so, what was the error it returned? (I'm **not** saying to do that now for the case of `git reset HEAD~`, I'm just trying to help you get more concrete feedback in general in the future :))

when I tried `git reset HEAD~` the command did not succeed. I had to exit the terminal manually by using `ctrl`+`c` . 

Then I tried resetting the entire project on my system from the very beginning and the `make` command succeeded , but the terminal's prompt didn't appear. I had to manually exit the terminal again. It seemed like there was some error with `source ~/.bashrc`. I tried solving this problem by replacing the `bashrc` file with a new one and that did work with fixing the terminal issue but it was still showing that more than 5000 files have been changed and are to be `git add` .
